src/epochs.lua

```diff
@@ -599,51 +586,55 @@ end
 --- 4. Increment the epoch stats for the gateways
 --- @param currentTimestamp number The current timestamp
 --- @return DistributedEpoch | nil # The updated epoch with the distributed rewards, or nil if no rewards were distributed
-function epochs.distributeRewardsForEpoch(currentTimestamp)
-	local epochIndex = epochs.getEpochIndexForTimestamp(currentTimestamp - epochs.getSettings().durationMs) -- go back to previous epoch
-	local epoch = epochs.getEpoch(epochIndex)
-	if not epoch then
+function epochs.distributeLastEpoch(currentTimestamp)
```

Comment:
In the unusual case that one or more entire epochs have been skipped, "last" might be a misnomer. Perhaps this should be parameterized by epochIndex rather than currentTimestamp?